xrviz/fields.py

```diff
@@ -107,6 +118,15 @@ def change_dim_selectors(self, *args):
             self._register(agg_selector, agg_selector.name)
             self.agg_selectors.append(agg_selector)
 
+    def setup_initial_values(self, init_params={}):
+        for widget in [self.x, self.y]:
+            if widget.name in init_params:
+                widget.value = init_params[widget.name]
+
+        for dim_agg in self.agg_selectors:
```

Comment:
Combine with line above, `for widget in [self.x, self.y] + self.agg_selectors:` ?

Reply:
This was my initial approach but we `can only concatenate list (not "Column") to list`, since `self.agg_selectors` is a `pn.Column`.

Reply:
`for widget in [self.x, self.y] + list(self.agg_selectors):`

Reply:
Ohh! my fault. That was simple. Thanks!